static/src/javascripts/projects/commercial/modules/dfp/dfp-env.js

```diff
@@ -15,7 +15,7 @@ define([
         adSlotSelector: '.js-ad-slot',
 
         /* prebidEnabled: boolean. Set to true if header bidding is enabled, and the user is not participating in the sonobi test */
-        prebidEnabled: config.page.edition == 'US' && !('tests' in config && config.tests.commercialHbSonobi),
+        prebidEnabled: config.page.edition == 'US' && !('tests' in config && config.switches.sonobiHeaderBidding),
```

Comment:
`config.page.edition == 'US' && !config.switches.sonobiHeaderBidding`, no? no need to check `config.tests` anymore.